linux/src/threadpool_linux.c

```diff
@@ -593,12 +689,10 @@ int threadpool_timer_start(THREADPOOL_HANDLE threadpool, uint32_t start_delay_ms
 
                 if (timer_delete(time_id) != 0)
                 {
-                    char err_msg[128];
-                    (void)strerror_r(errno, err_msg, 128);
-                    LogError("Failure calling timer_delete. Error: %d: (%s)", errno, err_msg);
+                    LogErrorNo("Failure calling timer_delete.");
                 }
+                (void)interlocked_exchange(&timer_instance->timer_status,  TIMER_DISABLED);
```

Comment:
>(void)interlocked_exchange(&timer_instance->timer_status,  TIMER_DISABLED); [](http://example.com/codeflow?start=16&length=75)

belongs after line 695 I think (in case timer_create fails)